### PR TITLE
SDT-129: CxfConfig added to sdt commissioning

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitBulkTest.java [hunk 1]
```diff
@@ -1,66 +1,63 @@
...
+import uk.gov.moj.sdt.ws._2013.sdt.bulkresponseschema.ObjectFactory;
+import uk.gov.moj.sdt.ws._2013.sdt.sdtendpoint.ISdtEndpointPortType;
+
+/**
+ * Test class for end to end web service tests..
+ *
+ * @author Robin Compston
+ */
+@ExtendWith(SpringExtension.class)
+public class SubmitBulkTest extends AbstractWebServiceTest<BulkRequestType, BulkResponseType> {
```
Comment (paulridout) [inline producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitBulkTest.java:21]: Does this class need to be annotated with `@SpringBootTest` like RequestBulkFeedbackTest has?

File: producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitQueryTest.java [hunk 2]
```diff
@@ -1,91 +1,89 @@
...
+import uk.gov.moj.sdt.ws._2013.sdt.submitqueryresponseschema.SubmitQueryResponseType;
+
+/**
+ * Test class for end to end web service tests..
+ *
+ * @author Robin Compston
+ */
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class SubmitQueryTest extends AbstractWebServiceTest<SubmitQueryRequestType, SubmitQueryResponseType> {
```
Comment (paulridout) [inline producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitQueryTest.java:23]: Does this class need to be annotated with `@SpringBootTest` like RequestBulkFeedbackTest has?

File: producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/CxfConfig.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    public ServletRegistrationBean<CXFServlet> cxfServlet() {
+        return new ServletRegistrationBean<>(new CXFServlet(), "/producers-comx/service/*");
+    }
+
+    @Bean(name = Bus.DEFAULT_BUS_ID)
+    public SpringBus springBus(LoggingFeature loggingFeature) {
+        SpringBus cxfBus = new SpringBus();
+        cxfBus.getFeatures().add(loggingFeature);
+        return cxfBus;
+    }
```
Comment (paulridout) [inline producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/CxfConfig.java:50]: Do we need this?  I might be wrong, but I don't think sdt-commissioning uses a queue.

Comment (kiran-yenigala-hmcts) [reply producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/CxfConfig.java:50]: Yes, we need this, this is not for service bus config

File: producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitQueryTest.java (hunk 2 above)
Comment (kiran-yenigala-hmcts) [reply producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitQueryTest.java:23]: it is annotated with @SpringBootTest

File: producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitBulkTest.java (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply producers-commissioning/src/end-to-end-test/java/uk/gov/moj/sdt/producers/comx/SubmitBulkTest.java:21]: added @SpringBootTest

**Pull request merged**